package.json

```diff
@@ -68,7 +76,12 @@
     "it-all": "^1.0.4",
     "it-drain": "^1.0.3",
     "it-last": "^1.0.4",
-    "uint8arrays": "^2.0.5"
+    "react-native-polyfill-globals": "^3.0.0",
+    "react-native-test-runner": "^2.3.0",
+    "react-native-url-polyfill": "^1.2.0",
+    "text-encoding": "^0.7.0",
+    "uint8arrays": "^2.0.5",
+    "web-streams-polyfill": "^3.0.1"
```

Comment:
why do we need this here ?

Reply:
We should be able to move this to RN test runner's config so it is installed in the test app directly instead. The same goes for `react-native-url-polyfill`.

Reply:
Sorry, I misunderstood your comment. 🙏 
Yes, we don't need web streams here but only in HTTP clien't test suite.

Reply:
maybe they can live in the `react-native-polyfill-globals` package

Reply:
I disagree. Users of `react-native-polyfill-globals` might want to polyfill only a few APIs. If all these dependencies live there, everything is pulled once the package is installed. If we don't want to pollute `js-ipfs-utils` dependency list with such dependencies, we can install them in the test app via its config.

Reply:
`web-streams-polyfill` and `react-native-url-polyfill` are now being installed in the test app.